### ISSUE TITLE
Add icon on media that lacks alt descriptions

### ISSUE THREAD

**Issue by dariusk** [hometown-fork/hometown#1165]
Mastodon now has a hint when you are drafting a media post that you need to add alt text. But there is still no indicator on other people's posts that the media lacks a description. Lots of people want to only boost described media.

My plan: add an icon to all media when it lacks a description. I choose this rather than what Twitter does (add an icon when there _is_ a description) because described media should be the default, "pretty" version of posts.

<hr>

Done! It looks like this, warning icon in the lower right:

![image](https://user-images.githubusercontent.com/266454/210295049-7a0d5a7d-5a9f-418a-857a-c2264d178401.png)

And similar for audio/video:

![image](https://user-images.githubusercontent.com/266454/210295076-ab36fd1e-b34f-42b5-99be-60b40fc66933.png)

**Comment by dariusk** [comment]
Thinking about this more -- it makes sense to also apply some classes so that admins can easily tweak this behavior if they don't like it specifically.

**Comment by dariusk** [comment]
Current WIP:

![image](https://user-images.githubusercontent.com/266454/209908720-67837e62-c097-4977-b92f-d6e8a352e21c.png)
